crates/forge/tests/cli/script.rs

```diff
@@ -1321,3 +1321,30 @@ contract SimpleScript is Script {
     let output = cmd.stdout_lossy();
     assert!(output.contains("ONCHAIN EXECUTION COMPLETE & SUCCESSFUL"));
 });
+
+// Asserts that running the same script twice only deploys library once.
+forgetest_async!(can_deploy_library_create2, |prj, cmd| {
```

Comment:
I'd suggest a second test that asserts the library is deployed once even when a different deployer account is used for the second script execution